src/main/java/com/microsoft/graph/models/extensions/Multipart.java

```diff
@@ -76,7 +86,7 @@ public void addHtmlPart(String name, String content) throws IOException {
 	}
 	
 	/**
-	 * Add an image part to the multipart body
+	 * Add an image/jpeg part to the multipart body
```

Comment:
Does OneNote only support adding jpegs? If so, let's rename this from addImagePart to addJpegPart. If it supports other types, we may want to consider adding an enum of supported image types.

Reply:
I don't think that this helps much more than just using the addFilePart so I'll just remove it

Reply:
👍